packages/cmf/src/bootstrap.md

```diff
@@ -22,6 +22,7 @@ cmf.bootstrap({
 | name | type | default | description |
 | -- | -- | -- | -- |
 | settingsURL | string | '/settings.json' | REQUIRED This URL to fetch the cmf settings.json file |
+| root | HTMLElement | document.getElementById(appId) | DOM element where to render the React application |
 | appId | string | 'app' | DOM element id, where to render the React application |
```

Comment:
What's the purpose of this appId if `root` is provided?

Reply:
it will not be ignored.